ci/docker-compose.ci.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3.7"
+
+services:
+  
+  trinitycore-ready:
+    image: tc-builder
+    command: >
```

Comment:
this is the test, only run in ci. Wait for all the ports, since it's in the same network as the other services